lib/reporters/ApiaryReporter.js

```diff
@@ -277,11 +277,12 @@ ApiaryReporter.prototype._transformTestToReporter = function _transformTestToRep
     duration: test.duration,
     result: test.status,
     startedAt: test.startedAt,
-    resultData: {
+    results: {
       request: test.request,
       realResponse: test.actual,
       expectedResponse: test.expected,
-      result: test.results,
+      validationResult: test.results,
+      errors: test.errors,
```

Comment:
If we had `test.errors || []` here, we can delete line 153 and if we had `test.results || {}`, we could delete line 154.

Reply:
Nice catch! Will do.